drivers/wifi/nrf700x/zephyr/src/qspi/src/qspi_if.c

```diff
@@ -111,6 +111,18 @@ struct qspi_nor_data {
 #endif /* CONFIG_MULTITHREADING */
 };
 
+static inline int qspi_freq_to_sckfreq(int freq)
+{
+#if defined(CONFIG_SOC_SERIES_NRF53X)
+	/* QSPIM (6-96MHz) : 192MHz / (2*(SCKFREQ + 1)) */
+	return ((192 / freq) / 2) - 1;
+#else
+	/* QSPIM (2-32MHz): 32 MHz / (SCKFREQ + 1) */
+	return (32 / freq) - 1;
```

Comment:
Is this supposed to be the same equation as noted above, i.e. (32MHz / (SCKFREQ + 1)) ?

Reply:
No, the comment is to derived QSPI frequency from serial clock frequency (SCK), but this function does reverse.